automation/vars/uni04delta-ipv6.yaml

```diff
@@ -0,0 +1,81 @@
+---
+vas:
+  uni04delta-ipv6:
+    stages:
+      - path: examples/dt/uni04delta-ipv6/control-plane/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+            --timeout=5m
+        values:
+          - name: network-values
+            src_file: values.yaml
+        build_output: nncp.yaml
+
+      - path: examples/dt/uni04delta-ipv6/control-plane
+        wait_conditions:
+          - >-
+            oc -n openstack wait osctlplane controlplane
+            --for condition=Ready
+            --timeout=60m
+        values:
+          - name: network-values
+            src_file: nncp/values.yaml
+          - name: service-values.yaml
+            src_file: service-values.yaml
+        build_output: control-plane.yaml
+
+      - path: examples/dt/uni04delta-ipv6/edpm-pre-ceph/nodeset
+        wait_conditions:
+          - >-
+            oc -n openstack wait
+            osdpns ceph-nodes --for condition=SetupReady
+            --timeout=10m
+        values:
+          - name: edpm-nodeset-values
+            src_file: values.yaml
+        build_output: nodeset-pre-ceph.yaml
+
+      - path: examples/dt/uni04delta-ipv6/edpm-pre-ceph
+        wait_conditions:
+          - >-
+            oc -n openstack wait
+            osdpd edpm-deployment-pre-ceph --for condition=Ready
+            --timeout=30m
```

Comment:
One thing that was recommended by the EDPM SMEs is to always wait on the `OpenStackDataPlaneNodeSet` but just wait for different conditions.  For instance, in the case of waiting for the associated `OpenStackDataPlaneDeployment` to finish, rather than waiting directly on that resource itself, you what for the `OpenStackDataPlaneNodeSet` to be `Ready`: https://github.com/openstack-k8s-operators/architecture/blob/1e2ed2bab6fc2a17d32aa7b620e66665dc9e3d18/automation/vars/default.yaml#L40-L45

Reply:
Ok, I copied this from `automation/vars/uni04delta.yaml` - do we want to create a separate PR change this accross the board instead of changing it here? A follow up?

Reply:
I think it's reasonable to address this in a follow up in both the IPv4 and IPv6 delta versions and not block this PR over it.